tools/owlbot-postprocess.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# Note: we could do various parts of this globally from the top level,
```

Comment:
Who calls this script?

Reply:
OwlBot itself. Probably easiest to explain in a VC :)